### DISCUSSION TITLE
images dont show up after uploading

### DISCUSSION THREAD

**Discussion by bym0** [outline/outline#2148] [Self Hosting]
I'm using a self hosted version, in a kubernetes cluster. Postgres, Minio, Redis.
This is very likely a problem on my side, logs dont say anything. Console also puts out no error. Somebody here with a similar problem or a solution? :)

**To Reproduce**
1. Create a new note
2. Drag and drop an image
3. Wait til the upload is done
4. Image should show up  

**Expected behavior**
Image should be visible  

**Screenshots**
![image](https://user-images.githubusercontent.com/6696609/119285030-6cd92b80-bc41-11eb-8c88-87e6997dfa21.png)  

**Outline (please complete the following information):**
- Install: self hosted
- Version: 028eb72f9cd8f698c9744020d6ddc13d897c11c0

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome 90.0.4430.212 & Vivaldi 3.8.2259.42

Best Regards and thank you!
Timo

**Comment by wenqvip** [comment]
it's not success when uploading in my side. I can not see the pic in minio
